Simple Incremental/Assets/Scripts/Projectile.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Projectile : MonoBehaviour
+{
+    Rigidbody2D rb2d = null;
+    SpriteRenderer spriteRenderer = null;
+    int damage = 0;
+    float falloffTime = 0f;
+    int maxPenetrations = 1;
+    float speed = 1f;
+    Queue<Projectile> projectiles = null;
+
+    private void Awake()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+        spriteRenderer = GetComponent<SpriteRenderer>();
+    }
+
+    private void Start()
+    {
+        projectiles = ProjectileManager.instance.projectiles;
+    }
+
+    public void Launch(Vector2 direction, Sprite _sprite, int _damage, float _falloffTime, int _maxPenetrations, float _speed)
```

Comment:
I don't like all these values being passed in separate, they really should be grouped together. If it was c I would say you need a struct. I'm not sure what the correct way in c# is to do that.

Reply:
It is a struct!

Good call. I'll go ahead and do that tonight.

Reply:
I'm going to hold off on structing this just yet. Reason being I might actually need those fields broken out to handle from a statssystem in the future, and I'm not sure how I'mm going to handle it yet.